source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -146,7 +146,7 @@ struct ZooKeeperProxyStats {
   ALL_ZOOKEEPER_PROXY_STATS(GENERATE_COUNTER_STRUCT)
 };
 
-enum class ErrorBudgetResponseType { FAST, SLOW };
+enum class ErrorBudgetResponseType { FAST, SLOW, NONE };
```

Comment:
I missed this before, but let's spell the enum values as `Fast`, `Slow`, `None`.

The style guide allows all-caps but that's problematic because that collides with the convention for `#define` and those are not scoped, so if you get a `#define NONE 0` pulled into your build then your code stops compiling. As this enum is new to this PR let's spell them without all-caps.